domains/jpsantistebanq.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jpsantistebanq",
+        "email": "jpsantq@gmail.com"
+    },
+    "record": {
+        "RECORDTYPE": "jpsantistebanq.github.io/portfolio/"
```

Invalid format. I guess you want CNAME, CNAME cannot have /

```suggestion
        "CNAME": "jpsantistebanq.github.io"
```